draft-kuehlewind-taps-crypto-sep.md

```diff
@@ -49,14 +49,14 @@ informative:
 
 --- abstract
 
-Due to the latency involved in connection setup and security handshakes, there is an increasing
-deployment of cryptographic session resumption mechanisms. While cryptographic context and
-endpoint capabilities need to be be known before encrypted application data can be sent, there is
-otherwise no technical constraint that the cryptographic handshake must be performed on the same
-transport connection. This document recommends a logical separation between the mechanism(s) used to negotiate
-capabilities and set up the initial encryption context as well as maintain this context (control protocol), 
-the application of encryption and
-authentication state to data (record protocol), and the associated transport connection(s).
+Secure transport protocols often consist of three logically distinct components: transport, 
+control (handshake), and record protection. This separation leads to modular protocol
+design and implementation. It is also increasingly common as cryptographic session resumption 
```

Comment:
From: "This separation leads..." to "...increase in use."

Replace with something like:

"Typically, such a protocol operates as a single module that is responsible for all three functions."